data/projects/m/metagov.yaml

```diff
@@ -11,24 +11,6 @@ social:
 github:
   - url: https://github.com/metagov
```

Comment:
The GitHub url(s) for metagov still need to be enumerated...
See my earlier comment:

> the other outstanding issue is that collision with the metagov entry that already exists. basically, our indexer assumes that anything that is contained with "github.com/metagov" belongs to metagov. So if it sees something like "github.com/metagov/daostar" then it will reject it (an artifact can only be claimed by one project). the best way to resolve this would be to enumerate any non-daostar github repos that should belong to metagov. alternatively, you keep everything in the metagov home.

here are a list of repos that we've already indexed. I'm guessing not all of these are relevant active. (this includes the ones you've already linked to daostar, so you can ignore those)

```
dao-governance-surfaces
lodestarfi-members-uri
daochem-website
govbase-website
grant-innovation-lab
constitution-template
governing-many-worlds
gateway
snapshot-hub
agreements-prototype
govarch-website
govbase
safefordaos
to-community
gateway-sandbox
open-web-challenge
discord-research-bot
gateway-website
d20-governance
daostar-explorer-backend
agreementengine
constitution-data
attestation-list-api
daostar-api
kms-tools
example-driver
extitute
interop
validator
veephd
eip-4824-proposals-uri-template
metagov.org
eip4824
daostar-deprecated
eip-4824-members-uri-template
commons
governance-archaeology
eips
attestation-list
reputable-contracts
dada-sourcecred
daostar-explorer-frontend
aragon-daouri-api
daostar-labs
discourse-web-monetization
proposalsuri
ai-constitution
sourcecred-instance
daostar-documentation
daostar
dao-uri-data-indexing-backend
cryptopolitics-paper
```